src/relay-runtime/src/RelayLogger.js

```diff
@@ -82,6 +82,13 @@ export default function RelayLogger(logEvent: LogEvent) {
     case 'execute.unsubscribe':
       logGroup(logEvent);
       break;
+    // New events, maybe add log function later
+    case 'store.notify.complete':
+    case 'store.notify.start':
+    case 'store.gc':
+    case 'store.restore':
+    case 'store.snapshot':
+    case 'store.publish': // End new events
```

Comment:
Feel free to remove the comments - I don't think they add much value.

Reply:
Ok, will do, I just wanted to point out that we might want to add some relay logger there, but I agree, I can remove the comment 👍